### ISSUE TITLE
Collection#update should have a variant that doesn't fail if record doesn't exist

### ISSUE THREAD

**Issue by glasserc** [Kinto/kinto.js#445]
```javascript
/**
 * Sets a record unconditionally.
 * This is like create() if the record doesn't exist or is deleted, or update() if the record already exists.
 */
set(record) {

}
```

This would simplify the use of Kinto.js as a key-value store.

**Comment by glasserc** [comment]
Actually, having looked at this for a while, I'm pretty sure this is the same as the `#update` operation. Closing.

**Comment by glasserc** [comment]
This is now called `put()` and was implemented as part of #455.
